TeslaSUI/TeslaSUI/Views/CustomStepper.swift

```diff
@@ -0,0 +1,32 @@
+// CustomStepper.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Кастомный степпер
+struct CustomStepper: View {
+    
+    // MARK: - Public properties
+        
+    @Binding var value: Int
+    
+    var body: some View {
```

Comment:
```suggestion
    var body: some View {
```
может разнести тоже в проперти?
но в такой маленькой вью не уверена

Reply:
по гайду не больше 50 строк)